src/lib/Chart.ts

```
@@ -974,13 +992,34 @@ export class Chart {
 
           break;
         case "dotted":
+          /** 요구사항에 따른  */
+          const areaGradientColorForDotted = this.createLinearGradient([
+            { offset: "0", stopColor: areaColor },
+            { offset: "1", stopColor: "rgba(255, 255, 255, 0)" },
+          ]);
```

ESLint

[error] Unexpected lexical declaration in case block.

(no-case-declarations)